requirements.txt

```diff
@@ -5,23 +5,26 @@ trio
 trio-typing
 
 # Docs
-mkdocs
 mkautodoc
+mkdocs
 mkdocs-material
 
 # Packaging
 twine
 wheel
 
 # Tests & Linting
-pytest
-pytest-asyncio
-pytest-trio
-pytest-cov
-black
 autoflake
-mypy
+black
+flake8
+flake8-bugbear
+flake8-pie
```

Comment:
I'm definitely +1 on flake8, but less keen on these extras, particularly `flake8-pie` is not very useful as 3/4 of its checks are for Celery. I know in the past @tomchristie has pushed back against using lots of extra linting tools.

Reply:
I added those to be consistent with `httpx`

https://github.com/encode/httpx/blob/655773e1c1b75895eda927d5a9d22a3b5b8f572d/requirements.txt#L21

Reply:
Ah I think I was misremembering when `flake8-comprehensions` got removed from httpx. To be clear, this is a minor nit. I'm happy with `flake8-bugbear`, but `flake8-pie` is literally one check ('PIE781: You are assigning to a variable and then returning. Instead remove the assignment and return.') which seems a bit unnecessary.